site/data/qasida/ya-alim-as-sir-minna.json

```
@@ -0,0 +1,484 @@
+
+            ["jūduk wa faḍluk wa birruk"],
+            ["yurjā wa baṭshuk wa qahruk"],
+            ["yukhshā wa dhikruk wishukrak"],
+            ["lāzim wa ḥamduk wal iijlāl"],
+
+            ["yā rabbi anta naṣīrī"],
+            ["falaqqinī kulla khayrī"],
+            ["wa jʿal jinānak maṣīrī"],
+            ["wa khtim bil iymānil 'ājāl"],
+
+            ["wa ṣalli fī kulli ḥālah"],
+            ["ʿalā muzīliḍ ḍalālah"],
+            ["man kallamathul ghazālah"],
+            ["muḥammadil haādiyil ddāl"],
+
+            ["wal ḥamdu ililh shukrā"],
+            ["ʿalā niʿam minhu tatrā"],
+            ["naḥmadhu sirran wa jahrā"],
+            ["wa bil ghadāyā wal 'āṣāl"]
+        ]
+    },
+    "translation": {
+        "en": [
+            ["O Knower of our inner-states,"],
+            ["do not expose us"],
+            ["Grant us ease and pardon us"],
+            ["and be with us wherever we are"],
+
+            ["My Lord, Knower of all states,"],
+            ["to you I direct all hopes"],
+            ["Bestow upon us your Divine attention,"],
+            ["be with us, and rectify the minds"],
+
+            ["My Lord, O Lord of Lords!"],
+            ["Your slave and impoverished one is at the door"],
+            ["He has come having severed all other means"],
+            ["seeking to make amends after having strayed"],
+
+            ["O Expansive in generosity, (we seek) Your generosity"],
+            ["All good is Yours and possessed by You"],
+            ["(that which is) beyond what Your slave could seek."],
+            ["So, repair with Your mercy right now"],
+
+            ["O Originator of all creation!"],
+            ["O Granter of all expansive kindness!"],
+            ["I ask that You cast down a veil"],
+            ["upon all foul actions and foolishness"],
+
+            ["O One who sees the reality of my heart!"],
+            ["Your awareness truly suffices me"],
+            ["So, erase with Your pardon my sin"],
+            ["and make right my intentions and deeds"],
+
+            ["My Lord! Upon You is my reliance"],
+            ["as well as my dependence is in You"],
+            ["- in truthfulness, and my furthest aim"],
+            ["is Your eternal sweet pleasure."],
+
+            ["My Lord, My Lord! Indeed I"],
+            ["ask You to pardon me"],
+            ["My opinion has never been at loss regarding You"],
+            ["O Owner of all dominion, O Guardian!"],
+
+            ["I complain to You, as I weep,"],
+            ["of the ills of my injustice and falsehood"],
+            ["and the evil of my deeds and my abandonment"],
+            ["and of my whimsical speech"],
+
+            ["and (I complain) of love of this blameworthy world"],
+            ["that is barren of any goodness"],
+            ["In it all tribulations reside,"],
+            ["and it is filled with tribulations and preoccupation"],
+
+            ["O Woe to my soul that decieves"],
```

```suggestion
            ["O Woe to my soul that deceives"],
```

(the error on the build is due to spelling of this)